### ISSUE TITLE
Docs: set the right repository on pub.dev

### ISSUE THREAD

**Issue by danielgomezrico** [martipello/hive_built_value#6]
**Is your feature request related to a problem? Please describe.**
Currently this link sents you to the original hive repository and not this one
<img width="1726" alt="image" src="https://github.com/user-attachments/assets/7f641b5a-3389-4294-9dbf-add9de3b1bde">

**Describe the solution you'd like**
It should open your repository: https://github.com/martipello/hive_built_value

It is pretty hard to find the code of the plugin

**Comment by martipello** [comment]
this is correct, it will be addressed if we ever manage to make a new release updating the analyzer and other packages
